code/game/mecha/equipment/weapons/weapons.dm

```diff
@@ -50,6 +52,7 @@
 		A.preparePixelProjectile(target, targloc, chassis.occupant, params, spread)
 
 		chassis.use_power(energy_drain)
+		projectiles--

```

Comment:
Some of the weapons fire in burst (3 shots) upon firing, but this will only decrement one.

Reply:
I tested it last night with the Seraph. It properly subtracted 6 per click when firing the machine gun on it. Check line 40, it's subtracting a projectile per projectile fired.

Reply:
Ah fair enough 👍